resources/views/pages/credits.twig

```diff
@@ -17,15 +17,10 @@
                 <h2>{{ __('Source code') }}</h2>
                 <p>{{ __('Version: _%s_', [version])|markdown }}</i></p>
                 <p>
-                    {{ __('The original engelsystem was written by
-[cookie](https://github.com/cookieBerlin/engelsystem).
-It was then completely rewritten and enhanced by [msquare](https://notrademark.de) (maintainer) and
-[MyIgel](https://myigel.name).')|markdown }}
+                    {{ __('The original engelsystem was written by [cookie](https://github.com/cookieBerlin/engelsystem). It was then completely rewritten and enhanced by [msquare](https://notrademark.de) (maintainer) and [MyIgel](https://myigel.name).')|markdown }}
```

Comment:
I hate those long lines but it seems that it can't be parsed otherwise (or are there any options for that?)

Reply:
Generally I prefer unique IDs like `credits.credit`, `credits.contributors` and so on instead of splitting them up more (which is imho useless here)